### DISCUSSION TITLE
Confusion about versions

### DISCUSSION THREAD

**Discussion by firesurfer** [kimai/kimai#4056] [Questions & Answers]
Hi everyone. We were just looking into upgrading our 1.17.1 installation to the the newest v2.x release and got a bit confused about the available versions.  We are running it inside a docker installation (https://hub.docker.com/r/kimai/kimai2)

When taking a look at the documentation one can find a migration guide from Kimai v1: https://www.kimai.org/documentation/migration-v1.html

But like the first sentence on the page says: 
> Be aware, Kimai 1 looks like that. If your Kimai looks different, you are not using Kimai 1 and do not need this guide! 

So our installation definitely looks different. In the about dialog it says:
![grafik](https://github.com/kimai/kimai/assets/8481806/392ee2ca-ffa8-482b-a64e-c3802238b8d9)

So the question in the end it:

Are we running kimai 1 or kimai 2 and how to identify it. 

And then the second question: In case we are running kimai 2 v1.17.1 (what I guess is the case) upgrading to v2.x we just need to follow the update steps provided in the docker container manual?

**Comment by kevinpapst** [comment]
> Are we running kimai 1 or kimai 2 and how to identify it.

Hahahaha ... I am sorry, but seriously 😁  The answer is obviously:

> If your Kimai looks different, you are not using Kimai 1 and do not need this guide!

For you and everyone else, who doesn't want to believe me I created another documentation page:
https://www.kimai.org/documentation/versions.html

Updates are explained here:
https://www.kimai.org/documentation/updates.html
